### PR TITLE
[fv.bea.tsaadane] Initial commit for fv-bea-tsaadane

### PR DESCRIPTION
**Pull request by dy2288**
Hi Keyman team!
Testing our process out with a model for Dane-Z̲aa Z̲áágéʔ, keyboard should be: https://keyman.com/keyboards/fv_dane_zaa_zaage

Hashes:

fv.bea.tsaadane.model.js
0ad792674ee8ff752aa9c353e88c1a4013b17bbb35636759fcca494d47886442  

fv.bea.tsaadane.model.kmp
2132bc5a1f137a48c3189654f7aa92bdee5ad352bc850c6986a51652fed1ea31  

fv.bea.tsaadane.model_info
aca69c5d168437cfaaa3c9b6fe85e36c1df71904a0ce971c357ae42292313be0

I wasn't sure what format to include these in the `release/fv/fv.bea.tsaadane/external_source` file.

### CODE REVIEW

File: release/fv/fv.bea.tsaadane/external_source [hunk 1]
```diff
@@ -0,0 +1,3 @@
+fv.bea.tsaadane.model_info=https://static.firstvoices.io/keyboards/lexical-models/fv.bea.tsaadane/fv.bea.tsaadane.model_info
+fv.bea.tsaadane.model.kmp=https://static.firstvoices.io/keyboards/lexical-models/fv.bea.tsaadane/fv.bea.tsaadane.model.kmp
+fv.bea.tsaadane.model.js=https://static.firstvoices.io/keyboards/lexical-models/fv.bea.tsaadane/fv.bea.tsaadane.model.js
```
Comment (mcdurdin) [inline release/fv/fv.bea.tsaadane/external_source:3]: Required format:

```suggestion
fv.bea.tsaadane.model_info https://static.firstvoices.io/keyboards/lexical-models/fv.bea.tsaadane/fv.bea.tsaadane.model_info aca69c5d168437cfaaa3c9b6fe85e36c1df71904a0ce971c357ae42292313be0
fv.bea.tsaadane.model.kmp https://static.firstvoices.io/keyboards/lexical-models/fv.bea.tsaadane/fv.bea.tsaadane.model.kmp 2132bc5a1f137a48c3189654f7aa92bdee5ad352bc850c6986a51652fed1ea31
fv.bea.tsaadane.model.js https://static.firstvoices.io/keyboards/lexical-models/fv.bea.tsaadane/fv.bea.tsaadane.model.js 0ad792674ee8ff752aa9c353e88c1a4013b17bbb35636759fcca494d47886442
```

**Review by mcdurdin — COMMENTED** [review]
This is looking good -- just need to make those corrections to external_source then we can do a test build.

**Comment by darcywong00** [comment]
Thank you for the submission.

I just skimmed the .model_info file and .kmp file. It seems the version is currently 
```json
      "description": "1.0.2022-03-10"
```

Is it possible to change the version to something like "1.0"?

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

**Comment by mcdurdin** [comment]
The correction to the format of the external_source file looks good. But I'm getting the following error report on the build:

```
[08:45:32][Step 3/3] Validating model /c/BuildAgent/work/25551df685ba4211/models/release/fv/fv.bea.tsaadane/
[08:45:34][Step 3/3] fv.bea.tsaadane.model_info: FAILED
[08:45:34][Step 3/3] sha256sum: WARNING: 1 computed checksum did NOT match
```

The sha256 I see for my downloaded copy of this file is `0e6de3866752e5865578d3419c3f1e6eb57c1b12ec3658123b608fbd2dc78ea5`. But I should note also that the sourcePath listed in the file is "experimental/fv/fv.bea.tsaadane" but should be "release/fv/fv.bea.tsaadane", so that's going to change the sha256 again.

**Comment by mcdurdin** [comment]
Thanks for the update; now I am seeing the .kmp has the wrong hash?

**Comment by mcdurdin** [comment]
Righto, this is building now; I can see one issue in the intermediate artifacts which I need to verify before we can merge (may require a separate PR and then a rebase on this PR to fix).

But as far as I can tell, nothing else required on your end; this is looking good!

**Comment by mcdurdin** [comment]
> I can see one issue in the intermediate artifacts

Fixed in https://github.com/First-Peoples-Cultural-Council/lexical-models/pull/1

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**